### PR TITLE
Bz#1797957

### PR DESCRIPTION
**Pull request by melcorr**
adding conditionals for operating systems
add snippet for firewall-cmd usage

### CODE REVIEW

File: guides/common/modules/proc_configuring-an-external-dhcp-server.adoc [hunk 1]
```diff
@@ -5,6 +5,8 @@
 To configure an external DHCP server to use with {ProductName}, on a Red{nbsp}Hat Enterprise Linux server, you must install the ISC DHCP Service and Berkeley Internet Name Domain (BIND) packages.
 You must also share the DHCP configuration and lease files with {ProductName}. The example in this procedure uses the distributed Network File System (NFS) protocol to share the DHCP configuration and lease files.
 
+include::snip_firewalld.adoc[]
```
Comment (tahliar) [inline guides/common/modules/proc_configuring-an-external-dhcp-server.adoc]: This include is outside the .Procedure markup, while the others are inside it. I don't mind which one you pick, but they should be consistent.

File: guides/doc-Provisioning_Guide/topics/Resources.adoc [hunk 2]
```diff
@@ -483,5 +483,10 @@ endif::[]
 # firewall-cmd --add-port=5900-5930/tcp
 # firewall-cmd --add-port=5900-5930/tcp --permanent
 ----
+ifeval::["{build}" == "foreman"]
```
Comment (tahliar) [inline guides/doc-Provisioning_Guide/topics/Resources.adoc]: Does this ifeval need to be here, since the file itself also contains an ifeval?

Comment (melcorr) [reply guides/doc-Provisioning_Guide/topics/Resources.adoc]: I need this ifeval to nest it with the ++ markup. Elseses it is a sentence that breaks the procedure. It's a bit like what Leo DiCaprio dealt with in that film Inception, I know, but with asciidoc, you take what is given to you. 

TL;DR: The + signs are the main object I need in that ifeval to frame the include.

Comment (tahliar) [reply guides/doc-Provisioning_Guide/topics/Resources.adoc]: Asciidoc strikes again!
